data-access/src/functions/http-graphql/schema/types/service-ticket.graphql

```diff
@@ -13,7 +13,35 @@ type ServiceTicket implements MongoBase {
   activityLog: [ServiceTicketActivityDetail]
   messages: [ServiceTicketV1Message]
   revisionRequest: ServiceTicketV1RevisionRequest
+  assignedVendor: VendorUser
+  id: ObjectID!
+  schemaVersion: String
+  createdAt: DateTime
+  updatedAt: DateTime
+}
+
+type VendorUserPersonalInformation {
```

Comment:
Remove all of these VendorUser graphql types, same reason as above, it's already defined